Gemfile

```diff
@@ -32,5 +32,7 @@ elsif ENV['RAILS_VERSION'] == 'edge'
   ENV['ENGINE_CART_RAILS_OPTIONS'] = '--edge --skip-turbolinks'
 elsif ENV['RAILS_VERSION']
   gem 'rails', ENV['RAILS_VERSION'], source: 'https://rubygems.org'
+else
+  Bundler.ui.warn '[Hyrax] Skipping all Rails dependency injection'
```

Comment:
i think this one can stay `warn`. it seems like we shouldn't reach this case normally?